### PR TITLE
fix(sweeper): sweep process

### PR DESCRIPTION
**Pull request by Monitob**

### CODE REVIEW

File: scripts/errcheck.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ echo "==> Checking for unchecked errors..."
 
 if ! which errcheck > /dev/null; then
     echo "==> Installing errcheck..."
-    go get -u github.com/kisielk/errcheck
+    go install github.com/kisielk/errcheck
```
Comment (remyleone) [inline scripts/errcheck.sh:8]:
```suggestion
    go install github.com/kisielk/errcheck@latest
```

File: scaleway/resource_vpc_public_gateway_pat_rule_test.go [hunk 2]
```diff
@@ -9,14 +9,6 @@ import (
 	vpcgw "github.com/scaleway/scaleway-sdk-go/api/vpcgw/v1beta1"
 )
 
-func init() {
```
Comment (remyleone) [inline scaleway/resource_vpc_public_gateway_pat_rule_test.go:12]: Why is this change needed?

File: scaleway/resource_vpc_public_gateway_dhcp_test.go [hunk 3]
```diff
@@ -9,13 +9,6 @@ import (
 	vpcgw "github.com/scaleway/scaleway-sdk-go/api/vpcgw/v1beta1"
 )
 
-func init() {
```
Comment (remyleone) [inline scaleway/resource_vpc_public_gateway_dhcp_test.go:12]: Why is this change needed?

File: scaleway/resource_vpc_gateway_network_test.go [hunk 4]
```diff
@@ -15,7 +15,7 @@ func init() {
 		Name: "scaleway_gateway_network",
 		F:    testSweepVPCGatewayNetwork,
 		// test depends upon PrivateNetwork, PublicGateway. Please add new resources for testing purpose.
-		Dependencies: []string{"scaleway_vpc", "scaleway_vpc_gateway_ip"},
```
Comment (remyleone) [inline scaleway/resource_vpc_gateway_network_test.go:18]: Why is this change needed?

File: scaleway/resource_rdb_database_test.go [hunk 5]
```diff
@@ -11,13 +11,6 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func init() {
```
Comment (remyleone) [inline scaleway/resource_rdb_database_test.go:14]: Why is this change needed?

File: scaleway/resource_rdb_user_test.go [hunk 6]
```diff
@@ -9,13 +9,6 @@ import (
 	"github.com/scaleway/scaleway-sdk-go/api/rdb/v1"
 )
 
-func init() {
```
[the remark this answers is not in the document]
Comment (Monitob) [reply scaleway/resource_rdb_user_test.go:12]: The call of this method was duplicated.

File: scaleway/resource_vpc_gateway_network_test.go (hunk 4 above)
Comment (Monitob) [reply scaleway/resource_vpc_gateway_network_test.go:18]: Absolutely the call of the sweep vpc was enough to remove the resource vpc. This was causing an error `already called`

File: scaleway/resource_vpc_public_gateway_dhcp_test.go (hunk 3 above)
Comment (Monitob) [reply scaleway/resource_vpc_public_gateway_dhcp_test.go:12]: IDEM, call duplicated

File: scaleway/resource_vpc_public_gateway_pat_rule_test.go (hunk 2 above)
Comment (Monitob) [reply scaleway/resource_vpc_public_gateway_pat_rule_test.go:12]: Idem, call duplicated

File: scaleway/resource_rdb_database_test.go (hunk 5 above)
Comment (Monitob) [reply scaleway/resource_rdb_database_test.go:14]: Idem, call duplicated.

**Review by remyleone — APPROVED** [review]
